docs/proposals/image-promotion.md

```diff
@@ -0,0 +1,201 @@
+# Image Promotion
+
+## Abstract
+
+This proposal describes the best practices and defines patterns for promoting Docker
+images between different stages (eg. from *development* to *production*).
+This proposal also describes a way how to configure OpenShift to serve as
+a Continuous Delivery tool.
+
+## Constraints and Assumptions
+
+This document describes the following ways to promote images:
+
+* **Based on human intervention**
+  * using the `oc` and `docker` commands manually
+* **Automated by the OpenShift platform**
+  * using enhanced *DeploymentConfig* [lifecycle hooks](https://docs.openshift.org/latest/dev_guide/deployments.html#lifecycle-hooks)
+  * using [ImageChangeTrigger](https://docs.openshift.org/latest/dev_guide/deployments.html#image-change-trigger)
+  * using [Projects](https://docs.openshift.org/latest/dev_guide/projects.html) for different "stages"
+* **Based on external Continuous Delivery tool**
+  * Jenkins
+  * [go.cd](http://go.cd)
+  * [snap](http://snap-ci.com)
+
+Docker images can be promoted (and the use-cases are valid for):
+
+* **Within a single Project**
+* **Between multiple Projects**
+* **Between multiple Clusters**
+
+The Docker image promotion can be done using the Docker [image tags](https://docs.docker.com/userguide/dockerimages/#setting-tags-on-an-image) or [labels](https://docs.docker.com/userguide/labels-custom-metadata/).
```

Comment:
Since labels is also k8s/OpenShift concept I'd suggest stating clearly you're talking about docker/image labels.